src/core/repositories/transactions.py

```
@@ -1,17 +1,28 @@
-        return self.filter_by_field('status', 'Uncategorized')
+    def all(self) -> list:
+        return list(TransactionsModel.objects.all().values())
+
+    def list_uncategorized(self) -> list:
+        return list(
+            TransactionsModel.objects.filter(status='Uncategorized').values())
+
+    def list(self) -> list:
+        return list(TransactionsModel.objects.all().values())
+
+    def get_by_id(self, id: str) -> dict:
```

Ruff (0.15.15)

[error] Function argument `id` is shadowing a Python builtin

(A002)